virt/release_notes/virt-4-14-release-notes.adoc

```diff
@@ -117,6 +117,9 @@ Deprecated features are included in the current release and supported. However,
 
 //CNV-29048 Release note: DEPRECATED FEATURE (Metrics backlog 4.14)
 
+//CNV-32032 Release note: DEPRECATED FEATURE (Windows 2012R2 templates deprecated)
+* Support for Windows Server 2012R2 templates is deprecated. 
```

Comment:
It looks like there is [supposed to be a space](https://www.microsoft.com/en-us/evalcenter/evaluate-windows-server-2012-r2):

```suggestion
* Support for Windows Server 2012 R2 templates is deprecated. 
```